src/sql/workbench/browser/modelComponents/text.component.ts

```diff
@@ -18,7 +18,13 @@ import { SafeHtml, DomSanitizer } from '@angular/platform-browser';
 @Component({
 	selector: 'modelview-text',
 	template: `
-		<p [style.width]="getWidth()" [innerHTML]="getValue()" [ngStyle]="this.CSSStyles" (click)="onClick()"></p>`
+		<div style="display:flex;flex-flow:row;align-items:center;" [style.width]="getWidth()">
+			<p [innerHTML]="getValue()" [ngStyle]="this.CSSStyles"></p>
+			<p  *ngIf="requiredIndicator" style="color:red;margin-left:5px;">*</p>
+			<div *ngIf="description" tabindex="0" class="modelview-text-tooltip" [attr.aria-label]="description">
+				<div class="modelview-text-tooltip-content" [innerHTML]="description"></div>
+			</div>
```

Comment:
Why are you doing this instead of just using the title/aria-label properties on the p element directly? I just added a tooltip to the text component which just does that so let's make sure we're doing the right thing here

Reply:
I did that and angular will remove it. this is the supported way I found.

Reply:
Remove what? Look at my PR - that seems to work just fine.

https://github.com/microsoft/azuredatastudio/pull/7312